playbooks/juju/cleanup.yaml

```diff
@@ -1,8 +1,13 @@
 - hosts: all
   tasks:
-    - name: 'tear down controller'
+    - name: 'force remove Juju machines'
+      command: juju remove-machine --force `seq 0 100`
+      failed_when: false
+    - name: 'destroy controller'
       command: >
-        juju remove-machine --force `seq 0 100`
         timeout 120 juju destroy-controller --destroy-all-models --destroy-storage -y {{ serverstack_cloud.region_name }}-{{ serverstack_cloud.region_name }}
```

Comment:
should it be 240 like in the other file?

Reply:
the cleanup job has a hard time limit at 5 minutes, so I want this to be a lot less forgiving on times